flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/observer/BaseObserver.java

```diff
@@ -103,10 +111,12 @@ protected void observeJmDeployment(
                 return;
             }
 
+            logger.info("JobManager is deployed but not ready yet", status);
```

Comment:
The old code seems more clear:
```
logger.info("JobManager deployment exists but not ready, status {}", status);
```
The new description `deployed but not ready yet` is not wrong but is too close to the enum `DEPLOYED_NOT_READY`, while we actully set the status to `DEPLOYING`.

Reply:
At this point it is always deploying status so no point in printing the current status. I could change it to 
```
JobManager is being deployed
```

Reply:
fixed